### PR TITLE
#12:Adding UI pages

### PR DESCRIPTION
**Pull request by pragatikanade**
Added below pages:
1)homepag.php (consider as header)
2)aboutus.php + aboutus.css (information about team)
3)addbook.php (form to add book details)
4)admindash.php (admin dashboard)
5)contactus.php  (contact information)
6)editbook.php (if admin wants to edit book info->edit and delete buttons)
7)style.scss (styling homepage,userdash,admindash)
8)userdash.php (user dashboard)

### CODE REVIEW

**Review by guptahemant — CHANGES_REQUESTED** [review]
Please squash the commits and add the issue number in front of commits.
Also in readme please include the steps needed to work with SCSS files and CSS for this project

**Comment by pragatikanade** [comment]
Squash done.
SCSS to CSS Comment in README.md done
Issued no infront of commit is done.

File: asset/css/login.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (guptahemant) [inline asset/css/login.css:86]: Remove this empty space please

File: README.md [hunk 2]
```diff
@@ -41,7 +41,9 @@ After install run the following command in the root. (this will install any depe
 ```bash
 $ composer install
 ```
-
+##SCSS to CSS
+whenever you want to change the CSS of Homepage.php you need to do changes in "style.scss" then compile this .scss file into style.css, as we 
+are adding style.css file in php.
```
Comment (guptahemant) [inline README.md:46]: Please document the tools / commands intended to be used for this compiling, Also i just noticed that SCSS file has been added for only one file, any reasons why it has not been added for others?

**Review by guptahemant — CHANGES_REQUESTED** [review]

**Review by guptahemant — APPROVED** [review]
For now we will proceed with this, @pragzii3896 please add a follow up to convert remaining CSS files to use SCSS and someone else can pick it up if they haven't worked on SCSS part.

**Pull request merged**